test/it/group-update.js

```diff
@@ -0,0 +1,40 @@
+const utils = require('../utils');
+const okta = require('../../');
+let orgUrl = process.env.OKTA_CLIENT_ORGURL;
+
+if (process.env.OKTA_USE_MOCK) {
+  orgUrl = `${orgUrl}/update-group`;
+}
+
+const client = new okta.Client({
+  orgUrl: orgUrl,
+  token: process.env.OKTA_CLIENT_TOKEN
+});
+
+describe('Group API tests', () => {
+  it('should update a group', async () => {
+    // 1. Create a new group
+    const newGroup = {
+      profile: {
+        name: 'Update Test Group'
+      }
+    };
+
+    const createdGroup = await client.createGroup(newGroup);
+    utils.validateGroup(createdGroup, newGroup);
+
+    // 2. Update the group name and description
+    const updateGroup = {
+      profile: {
+        name: 'Update Test Group - Updated',
+        description: 'Description updated',
+      }
+    };
+
+    const updatedGroup = await client.updateGroup(createdGroup.id, updateGroup);
```

Comment:
This is interesting. We're not updating the `createdGroup` and resending it. That means our tape will not include the `id` property in the expected body. Some SDK's, like the PHP SDK, may not allow sending a model without an ID, so those SDKs couldn't pass this test on the mock-okta server.

We should modify `createdGroup` with the new `name` and `description`, then send that.